lib/linter.js

```diff
@@ -206,3 +208,24 @@ function getMessage(input, message) {
   const formatted = ((typeof message) === 'function') ? message(input) : message;
   return formatted.trim();
 }
+
+function lintWebpack() {
+  if (fs.existsSync('./webpack.config.js')) {
+    console.error(stripIndents`
+      This project contains a webpack.config.js file. Depending on your needs
+      this might not be necessary with A3.
+
+      Frontend scripts can be placed in the ui/src/index.js file of the module or
```

Comment:
Since we're linking to docs elsewhere we might be better off linking to the guide instead of this. It's a good encapsulation, but we have a whole guide we can lean on already that's comprehensive.